docs-ref-conceptual/get-started-tutorial-1-prepare-environment.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Configure your Azure CLI environment | Microsoft Docs
+description: Learn Azure CLI installation, sign in, setting environment variables, and creating Azure resources containing a random ID.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, az account, az group, az config, az init, configuration
+---
+# Prepare your environment for the Azure CLI
+
+In this tutorial step, you execute your first Azure CLI commands! This includes finding and setting your active subscription and setting default values. You also learn how to create resource groups containing a random ID to always guarantee a unique resource name.
+
+## Install the Azure CLI
+
+Get started by first selecting your environment.
+
+* If you don't have an [Azure subscription](/azure/guides/developer/azure-developer-guide#understanding-accounts-subscriptions-and-billing), create an [Azure free account](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio) before you begin.
+
+* Use the Bash environment in [Azure Cloud Shell](/azure/cloud-shell/overview) by selecting the **Open Cloud Shell** button in the top right corner of each Azure CLI code block.
+
+* If you prefer to run the Azure CLI reference commands locally, [install](/cli/azure/install-azure-cli) the Azure CLI.
+
+* To complete the steps in this tutorial, you need `contributor` permissions in your subscription.
+
+[!INCLUDE [include](./includes/current-version.md)]
+
+## Sign in to Azure using the Azure CLI
+
+There are several [authentication options](./authenticate-azure-cli.md) when working with the Azure CLI. The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+
+[!INCLUDE [interactive_login](includes/interactive-login.md)]
+
+## Find and change your active subscription
+
+After logging into the Azure CLI, always check your current subscription. If you aren't working under the subscription you prefer, change your subscription using [az account set](/cli/azure/account#az-account-set). Here's a code example to use.
+
```

Comment:
I think there should be note stating that "Make sure to replace <yourSubscriptionName> and <yourSubscriptionID> with the actual name or ID of the subscription you want to switch to."

Reply:
That's what `<>` is supposed to indicate.  In Azure CLI code blocks, `<>` is colored in red.